internal/controller/driver_controller.go

```diff
@@ -741,7 +831,22 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 								),
 							})
 						}
-
+						// CSI LogRotate Container
+						if logRotationEnabled {
+							resources := ptr.Deref(pluginSpec.Resources.LogRotator, corev1.ResourceRequirements{})
+							containers = append(containers, corev1.Container{
+								Name:            "log-rotator",
+								Image:           r.images["plugin"],
+								ImagePullPolicy: imagePullPolicy,
+								Resources:       resources,
+								Command:         []string{"/bin/bash", "-c"},
+								Args:            []string{`while true; do logrotate --verbose /logrotate-config/csi; sleep 15m; done`},
```

Comment:
There is no need for command and arg. Just have one command and move it into a const to be reused.